tests/feature_elimination/test_feature_elimination.py

```diff
@@ -337,6 +339,75 @@ def test_get_feature_shap_values_per_fold(X, y):
     assert train_score > 0.9
     assert shap_values.shape == (2, 3)
 
+def test_shap_rfe_same_features_are_kept_after_each_run():
+    """
+    Test a usecase which appears to be flickering with probatus 1.8.6 and lower.
+
+    Expected result: every run a different outcome.
+    """
+    SEED = 1234
+
+    feature_names = [
+        "f1",
+        "f2",
+        "f3_static",
```

Comment:
why "static" and in "f3_static"? 

What about

```
feature_names = [f"f{num}" for num in range(1, 21}]
```

with eventually a rename if you need "f3_static"

Reply:
Fixed.